op-challenger/game/fault/solver/game_solver.go

```diff
@@ -50,7 +52,7 @@ func (s *GameSolver) CalculateNextActions(ctx context.Context, game types.Game)
 }
 
 func (s *GameSolver) calculateStep(ctx context.Context, game types.Game, agreeWithRootClaim bool, claim types.Claim) (*types.Action, error) {
-	if claim.Countered {
+	if claim.CounteredBy != common.BigToAddress(big.NewInt(0)) {
```

Comment:
Is this just `common.Address{}` or does it come out with something different?

Reply:
This actually doesn't compile :(

Reply:
ugh, sorry, go is a bit daft...
```suggestion
	if claim.CounteredBy != (common.Address{}) {
```